packages/katello/katello/katello.cron

```diff
@@ -0,0 +1,12 @@
+SHELL=/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+
+# Script for intiating removal of orphaned content
+00 22 * * 0	root	foreman-rake katello:delete_orphaned_content RAILS_ENV=production >/dev/null 2>&1
+
+# cronjob to clean puppet envs for puppet 4
+15 22 * * 0	root	[ -d /etc/puppetlabs/code/environments ] &&  find /etc/puppetlabs/code/environments/KT* -maxdepth 0 -type d -empty -delete > /dev/null 2>&1
```

Comment:
Isn't the find equivalent to `rmdir --ignore-fail-on-non-empty /etc/puppetlabs/code/environments/KT*`? I'm fine with keeping it the current way to keep the change small though.

Reply:
Very possible, but would prefer not to change this to keep the change small and limit the scope of the BZ as well.